lib/rubygems/executor.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+require 'open-uri'
+require 'json'
+require 'rubygems/platform'
+
+module Gem
+  module Web
+    class Executor
+
+      def supported_options_from_metadata
+        ["source_code_uri", "documentation_uri"]
+      end
+
+      def open_page(gem, options)
+        spec = Gem::Specification.find_by_name(gem)
```

Comment:
```ruby
begin       
  spec = Gem::Specification.find_by_name(gem)
rescue Gem::MissingSpecError => e
   puts e.message
end
```

Reply:
How should I prevent running the remaining code? Adding a `exit 1` after the error message?